src/main/resources/io/jenkins/docker/connector/DockerComputerAttachConnector/config.jelly

```diff
@@ -10,7 +10,7 @@
             <li>
                 Entrypoint must be able to accept jenkins slave connection parameters.
                 See 
-                <a href="https://github.com/jenkinsci/docker-jnlp-slave">jenkins/docker-jnlp-slave</a>
+                <a href="https://github.com/jenkinsci/docker-jnlp-slave">jenkins/jnlp-slave</a>
```

Comment:
I think that, if we're going to update this text, it'd be best if the link-text matched the link-URL so, given that the source code uses a different name to the docker container, it'd be better if we linked to both (that also means that, if things change again, users have twice the chance of tracking down the new location).

So I think we should say something like:
> See docker container [jenkins/jnlp-slave](https://hub.docker.com/r/jenkins/jnlp-slave/) and/or source [jenkinsci/docker-jnlp-slave](https://github.com/jenkinsci/docker-jnlp-slave).